pkg/gcp/actuator/actuator.go

```diff
@@ -555,10 +555,15 @@ func (a *Actuator) updateProviderStatus(ctx context.Context, logger log.FieldLog
 	return nil
 }
 
+// GetParentCredSecretLocation returns the namespace and name where the parent credentials secret is stored.
+func (a *Actuator) GetParentCredSecretLocation() types.NamespacedName {
+	return types.NamespacedName{Namespace: constants.KubeSystemNS, Name: annotatorconst.GCPCloudCredSecretName}
```

Comment:
global var?

Reply:
Can you clarify? I'm not quite sure of the benefit there.

Reply:
Ignore this. This was a set of comments I was making before I saw that you extended the actuator API to include GetParentCredSecretLocation(), and this must have slipped through as I removed those comments.

Reply:
Gotcha, thanks.